src/Installers/Windows/WindowsHostingBundle/DotNetCore.wxs

```diff
@@ -12,7 +24,7 @@
                         InstallCondition="VersionNT64 AND (NOT OPT_NO_RUNTIME OR OPT_NO_RUNTIME=&quot;0&quot;)"
                         InstallCommand="/quiet /norestart"
                         RepairCommand="/quiet /repair"
-                        Permanent="yes"
+                        UninstallCommand="/quiet /uninstall"
```

Comment:
I would consider adding `/norestart`

Reply:
`Permanent="yes"` sounds very intentional. What are we _losing_ by enabling uninstall❔

Reply:
Nothing AFAIK. It was originally authored this way. This was likely to deal with things like SxS installs of VS2015 (which included ANCM for IIS). We later added ANCM for VS 2017 so it's possible this was related to avoiding removing the runtimes in SxS scenario's since they did not support bundle-to-bundle reference counts

Reply:
To be clear, I signed off but defer to you -- @joeloff. We need you for the **real** approval for both your expertise and your request for changes.

Reply:
@wtgodbe perhaps I'm looking at a cached page. The part of this conversation about `Permanent="yes"` has defnitely been resolved. Did you add `/noRestart`❔

Reply:
I haven't, I don't think it's necessary for this PR